src/index.ts

```diff
@@ -11,5 +11,8 @@ export {
 export {
   WithinContentContext as _SECRET_INTERNAL_WITHIN_CONTENT_CONTEXT,
 } from './utilities/within-content-context';
+export {
+  AppBridgeContext as _SECRET_INTERNAL_APP_BRIDGE_CONTEXT,
```

Comment:
I still don't feel the best about exposing this but if we're removing it in `v5` is not too big of a deal, I just don't want us to get in the habit of doing this 😬

Reply:
I guess the alternative is we say we're not going to expose this, and that people need to migrate to app-bridge-react before they move to v4

Reply:
I also feel like that approach is a little off since we're deprecating app-bridge. I think this will be ok if we keep this to a minimal in the future 😄

Reply:
Given that we the appBridge context was intended to be public, and we document it and that it is required for people that want to use app-bridge I think this should have a regular name `AppBridgeContext` rather than a scary "if you need to do this then you're probably doing something wrong" name.


For contrast The WithinContentContext was never documented or intended to be public in v3, and we're only exporting it as `_SECRET_INTERNAL_WITHIN_CONTENT_CONTEXT` because some corners of web need it, and we've not yet got a plan for how to expose this in a more controlled manner.